db/tikv/db.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/magiconair/properties"
 	"github.com/pingcap/go-ycsb/pkg/ycsb"
 	"github.com/pingcap/tidb/config"
```

Comment:
em, why still use tidb config here?

Reply:
`client-go` does not support coprocessor so we still need tidb now.

Reply:
oh, sadly, maybe it is better to remove coprocessor support 😭 

PTAL @AndreMouche

Reply:
If we don't use coprocessor, I think it is ok to remove it.